Seemingly the only breaking change in these updates is dropping support
for old Python releases, so we should be fine.

--- setup.py
+++ setup.py
@@ -16,8 +16,8 @@ install_requirements = [
     'click >= 4.1',
     'Pygments >= 2.0',  # Pygments has to be Capitalcased. WTF?
     'prompt_toolkit>=2.0.6,<2.1.0',
-    'psycopg2 >= 2.7.4,<2.8',
-    'sqlparse >=0.2.2,<0.3.0',
+    'psycopg2 >= 2.7.4,<2.9',
+    'sqlparse >=0.2.2,<0.4.0',
     'configobj >= 5.0.6',
     'humanize >= 0.5.1',
     'cli_helpers[styles] >= 1.2.0',
